index.d.ts

```diff
@@ -0,0 +1,22 @@
+import inquirer, { Answers } from 'inquirer';
+
+declare module 'inquirer' {
```

Comment:
To extend the inquirer interfaces, this seems more appropriate to get alignment:
```suggestion
declare namespace inquirer {
```
See https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/inquirer/index.d.ts#L95

Note: So far, I have not typed across packages myself, so I may be missing something. This comment is really based only on my observation in `@types/inquirer`.